model/update_salts.py

```diff
@@ -0,0 +1,129 @@
+import os
+import requests
+from urllib import request
+from multiprocessing import Pool
+from functools import partial
+import xml.etree.ElementTree as ET
+
+
+DIR_PATH = os.path.dirname(os.path.realpath(__file__))
+FILE_PATH = 'ftp://ftp.ebi.ac.uk/pub/databases/chebi/ontology/nightly/chebi.obo'
+N_PROCESSES = 20
+
+
+def create_salts_mapping():
+    with request.urlopen(FILE_PATH) as f:
+        print('Retrieving the CHEBI file...')
+        all_chemicals = [i for i in f.read().decode("iso-8859-1").split('\n\n') if i[:5] == '[Term']
+        print('File is ready')
+        chebi_to_smiles = identifier_mapping(all_chemicals, 'smiles')
+        chebi_to_inchi = identifier_mapping(all_chemicals, 'inchi')
+        smiles_to_chebi = revert_mapping(chebi_to_smiles)
+        ###
+        smiles_to_chebi['[NH4+]'] = [28938]
```

Comment:
Consider commenting why this hardcoded magic mapping is needed

Reply:
couldn't find the good match automatically for this metabolite, so since it's one of the important ones it should get in in some form